documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -3,7 +3,6 @@
 cb.marshaller(new org.infinispan.commons.marshall.ProtoStreamMarshaller())
   .statistics()
       .enable()
-      .jmxDomain("org.infinispan")
```

Comment:
jmxDomain is not deprecated in HR client. This line can be kept, but maybe I would use something different than "org.infinispan", which is the default anyway, so it's not useful.